code/scripting/api/libs/graphics.cpp

```diff
@@ -2179,5 +2180,31 @@ ADE_FUNC(freeAllModels, l_Graphics, nullptr, "Releases all loaded models and fre
 	return ADE_RETURN_NIL;
 }
 
+ADE_FUNC(createColor,
```

Comment:
Making the alpha value optional (defaulting to fully opaque) would be a nice QOL enhancement.